SHIBBOLETH_SP.md

```diff
@@ -6,27 +6,45 @@ category: authentication
 
 This document describes installing Shibboleth on Debian 9.
 
+## Requirements
+
+1. Apache
+
+        Apache webserver is required for Shibboleth SP.
+
+2. Root Access
```

Comment:
`sudo` implies that you have the ability to run commands as "root", so not sure what the value of this is?

Reply:
removed